Modules/ThirdParty/HDF5/src/itkhdf5/src/H5make_libsettings.c

```diff
@@ -43,154 +38,143 @@ static const char *FileHeader = "\n\
 
 #define LIBSETTINGSFNAME "libhdf5.settings"
 
+FILE       *rawoutstream = NULL;
+
 
 /*-------------------------------------------------------------------------
- * Function:	insert_libhdf5_settings
- *
- * Purpose:	insert the contents of libhdf5.settings into a file
- *		represented by flibinfo.
- *		Make it an empty string if H5_HAVE_EMBEDDED_LIBINFO is not
- *		defined, i.e., not enabled.
+ * Function:  insert_libhdf5_settings
  *
- * Return:	void
- *
- * Programmer:	Albert Cheng
- *		Apr 20, 2009
+ * Purpose:   insert the contents of libhdf5.settings into a file
+ *            represented by flibinfo.
+ *            Make it an empty string if H5_HAVE_EMBEDDED_LIBINFO is not
+ *            defined, i.e., not enabled.
  *
+ * Return:    void
  *-------------------------------------------------------------------------
  */
 static void
 insert_libhdf5_settings(FILE *flibinfo)
 {
-#if defined(H5_HAVE_EMBEDDED_LIBINFO) && !defined(__EMSCRIPTEN__)
-    FILE *fsettings;	/* for files libhdf5.settings */
+#ifdef H5_HAVE_EMBEDDED_LIBINFO
```

Comment:
Is removal of emscripten-related additions intentional? It also appears in a few other places.

Reply:
Good catch. Reviewing / adding...